yb-voyager/cmd/liveMigrationReportCommand.go

```diff
@@ -59,7 +63,7 @@ var liveMigrationReportCmd = &cobra.Command{
 			}
 			if migrationStatus.FallbackEnabled {
 				getSourceDBPassword(cmd)
-				migrationStatus.SourceDBAsTargetConf.Password = source.Password
+				migrationStatus.SourceDBAsTargetConf.Password = tconf.Password
```

Comment:
curious: Is this a bug fix, essentially? because source.Password would have been empty then, right?